t/03-testapi.t

```diff
@@ -85,6 +85,10 @@ sub fake_read_json {
     elsif ($cmd eq 'backend_mouse_set') {
         return {ret => {x => 100, y => 100}};
     }
+    elsif ($cmd eq 'backend_type_string') {
+        diag explain $cmd;
+        return  $cmds;
+    }
```

Comment:
I assume you only added this for debugging? Should be removed again when you are done

Reply:
Actually we were wondering if we had to mock `backend_type_string` your comment further down, clarifies this!